README.md

```diff
@@ -1,70 +1,69 @@
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Preview
+![screenshot](./images/screenshot.png)
 
-## Available Scripts
+## Project Presentation
 
-In the project directory, you can run:
+- [Presentation](https://drive.google.com/file/d/1x8L418Dmo94-Zcl_GiO9QlTXD9oiZ3x7/view?usp=sharing)
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- HTML, CSS and Javascript
+- React
+- Redux
+- Github
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Link
 
-### `npm test`
+[Click here](https://deploy-preview-3--monumental-paletas-7ab791.netlify.app)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+### Setup
+~~~ 
+git clone https://github.com/Smart1-hub/weather-api
+cd weather-api
 
-### `npm run build`
+~~~
+- install project dependencies
+```
+npm install
+```
+- start project
+```
+npm start
+```
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Environment Prerequisites
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+Web Browser
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Code editor: Vscode or any other code editor
 
-### `npm run eject`
+## Authors
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+👤 **Ademola Adebayo**
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+- Github: [@Smart1-hub](https://github.com/Smart1-hub)
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+- Twitter: [@ademola_adebayo](https://twitter.com/ademola_adebayo)
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+- LinkedIn: [Ademola Adebayo](https://www.linkedin.com/in/ademola-adebayo-81051578/)
 
-## Learn More
+## Acknowledgement
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- Original design idea is by Nelson Sakwa in Behance.
```

Comment:
✅ The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) 👍🏻